### PR TITLE
Features/movie page stylling

### PR DESCRIPTION
**Pull request by WojciechBylica**
I've set together the results of our Sprint.
changes:removed white spaces  and not working paddings
added margin-bottom

### CODE REVIEW

File: src/features/Tiles/PeopleTiles/styled.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import styled from "styled-components";
 
 export const SectionWrapper = styled.section`
-    padding: 0 16px;
```
Comment (KarolCiesluk) [inline src/features/Tiles/PeopleTiles/styled.js:4]: Now we don't need this component any more so maybe delete it and write plain `<section>` in index.js

File: src/features/Tiles/BigMovieTile/styled.js [hunk 2]
```diff
@@ -1,24 +1,29 @@
...
-        padding: 16px; 
+        padding: 16px;
         display: grid;
         grid-template-columns:  min-content 1fr;
         grid-template-rows: auto;
         grid-gap: 16px;
-        grid-template-areas: 
+        grid-template-areas:
         "image content"
         "description description";
```
Comment (KarolCiesluk) [inline src/features/Tiles/BigMovieTile/styled.js:28]: perhaps indentation is needed here

**Review by KarolCiesluk — CHANGES_REQUESTED** [review]
Hey! Good idea to tidy it up! Just be careful with deleting stuff :P  And the same question as on trello: Do we still need: `<div>Popular Movies</div>` between Navigation and Header?

File: src/features/Tiles/BigMovieTile/styled.js [hunk 3]
```diff
@@ -1,24 +1,29 @@
 import styled from "styled-components";
 
 export const MovieDetailsTile = styled.article`
-    padding: 40px;
```
[the remark this answers is not in the document]
Comment (WojciechBylica) [reply src/features/Tiles/BigMovieTile/styled.js:4]: i added it again: padding 40px and on medium screen 32px added

File: src/features/Tiles/PeopleTiles/styled.js (hunk 1 above)
Comment (WojciechBylica) [reply src/features/Tiles/PeopleTiles/styled.js:4]: ok - changed for random section

File: src/features/Tiles/BigMovieTile/styled.js (hunk 2 above)
Comment (WojciechBylica) [reply src/features/Tiles/BigMovieTile/styled.js:30]: done.

**Review by KarolCiesluk — APPROVED** [review]
Looks good to me! Well done :)

**Review by patrycja-i-lesniak — APPROVED** [review]
It's nice to see the effects of our work. Good job.

**Pull request merged**